doc/conf.py

```diff
@@ -54,14 +43,15 @@ def __getattr__(cls, name):
     'sphinxcontrib.bibtex',
     'edit_on_github',
     'sphinx_autodoc_typehints',
-    'nbsphinx',
-    'sphinx_gallery.gen_gallery',
     "sphinx.ext.intersphinx",
     "sphinx_automodapi.automodapi",
     'sphinx_copybutton',
     "m2r"
 ]
 
+
+intersphinx_mapping = {"photonics": ("http://sfdocsredesign.s3-website-us-east-1.amazonaws.com/photonics", None)}
```

Comment:
Was making the assumption that no one will go looking in the deep dark depths of this PRs changed files, so I might delete this comment :) This was required so that intersphinx would work properly, but this will be changed to point to sf.ai before this is merged.